src/main/java/sg/ncl/MainController.java

```diff
@@ -60,8 +60,10 @@
 
     private String AUTHORIZATION_HEADER = "Basic dXNlcjpwYXNzd29yZA==";
 
+    private final String contactEmail = "support@ncl.sg";
+
     // error messages
-    private final String ERR_SERVER_OVERLOAD = "There is a problem with your request. Please contact support@ncl.sg";
+    private final String ERR_SERVER_OVERLOAD = "There is a problem with your request. Please contact " + contactEmail;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Make this final field static too. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1170)